### ISSUE TITLE
Okteto CLI is being reported as a virus in Windows

### ISSUE THREAD

**Issue by rberrelleza** [okteto/okteto#2443]
**Describe the bug**
![image](https://user-images.githubusercontent.com/475313/160494183-a19c96a4-5577-4a07-bc91-c9a138d9974b.png)

**Desktop (please complete the following information):**

- OS: Windows

**Additional context**
I'm waiting on the user to confirm the windows version and installation method

**Comment by AgustinRamiroDiaz** [comment]
I've downloaded the cli from https://downloads.okteto.com/cli/okteto.exe and ran the antivirus on the particular binary. I couldn't reproduce the issue

**Comment by rberrelleza** [comment]
No, but we've been getting reports of this every now and then. I don't know enough of Windows binary packaging to recommend a solution.

**Comment by AgustinRamiroDiaz** [comment]
@rberrelleza @jmacelroy maybe we could change our default recommendation of installation to be scoop (or chocolately in the future)?

**Comment by rberrelleza** [comment]
I don't know enough of the windows ecosystem to know how common is it to use scoop or chocolatey. Is this common in other big windows tools?

**Comment by AgustinRamiroDiaz** [comment]
@rberrelleza I don't know if you are asking if it's common to use scoop/chocolatey or the antivirus issue, so I'll answer both

Scoop/chocolatey have gained a lot of traction in windows development (scoop is more liked but chocolatey is from Microsoft). I think these should be the preferred ways of installation since it handles downloading binary + adding it to path + managing upgrades. This work is tedious with the "binary installation"

On the other hand, the windows antivirus is not great. It has a lot of false positives (and negatives) that lead to this warnings. Even though, we should be able to pass the antivirus test.
